### ISSUE TITLE
GitHub repo link on https://docs.solo.io/gloo doesn't come to this repo

### ISSUE THREAD

**Issue by ahmetb** [solo-io/solo-docs#656]
It goes to https://github.com/solo-io/gloo repo which is not where these docs are hosted, so that's confusing. I hope you consider fixing.

**Comment by mitchdraft** [comment]
Thanks for raising this concern, we want to make it easy to contribute to the docs and code.

We have moved the Gloo docs to the gloo repo and will gradually do the same with our other projects in this repo.

Deprecation notice: https://github.com/solo-io/solo-docs/tree/master/gloo
New home of docs: https://github.com/solo-io/gloo/tree/feature-rc1/docs (note the branch - which I now see adds to the confusion. It will merge to master soon)

Closing as the current link is correct, as will be clear when feature-rc1 merges.
